src/libsync/account.cpp

```diff
@@ -463,7 +463,7 @@ bool Account::serverVersionUnsupported() const
         // not detected yet, assume it is fine.
         return false;
     }
-    return serverVersionInt() < makeServerVersion(7, 0, 0);
+    return serverVersionInt() < makeServerVersion(9, 1, 0);
```

Comment:
I agree about raising this because we do in fact not test the client against these old server versions much.

How did you chose 9.1?

Reply:
As explained in the commit message:  because it's end of life of 9.0 is reached.